### ISSUE TITLE
Android wrapping.

### ISSUE THREAD

**Issue by samburgers** [10up/flexibility#46]
Just wondering if this is designed (or roadmapped) to fix the wrapping bug in Android browser 4 -> 4.3, and UC Browser 9.9 for Android? 

I have attempted to use it to polyfill those browser to no success. 

Many thanks

http://caniuse.com/#feat=flexbox
"Only supports the old flexbox specification and does not support wrapping."

**Comment by jonathantneal** [comment]
TL;DR; yes it is roadmapped for the next major release.

The next version would allow you to add flexibility to browsers outside of IE. Right now, I am leveraging a proprietary style in IE called `currentStyle`, which is why this script only works for IE. However, this script could be extended in a manner that allows you to use it for Android. In this case, you would end up needing something like an additional build process that tells JavaScript which elements are flex, or alternatively, you would need to rely on something like data attributes to tell the script which elements are flex. In short, unless I know which elements are flex (like I do using the aforementioned IE proprietary hack), then I need to be told explicitly.

**Comment by samburgers** [comment]
great to hear, and thanks for the reply.

**Comment by samburgers** [comment]
Just wondering if this ever make it into 2.x? still appears to be an issue on Android 4.2 - 4.4 and possibly others.

**Comment by afilp** [comment]
I wonder the same too, is this fixed?
